### ISSUE TITLE
Juniper configuration-less ports are not in the interfaces list

### ISSUE THREAD

**Issue by lindycoder** [internap/netman#25]
A juniper with no configuration on a port will not show it when querying interfaces via the netconf protocol

So the /interfaces should look for an official list

There is a CLI command : show interfaces terse
That will show all interfaces names, we need to find how to do it the netconf-way

**Comment by mat128** [comment]
See the chapter on Requesting Operational Information Using NETCONF
https://www.juniper.net/techpubs/en_US/junos12.3/information-products/topic-collections/netconf-guide/netconf-guide.pdf

Basically:
```
<rpc>
<get-interface-information>
<brief/>
<interface-name>ge-0/3/0</interface-name>
</get-interface-information>
</rpc>
```

I assume the following also works (to be tested):
```
<rpc>
<get-interface-information>
<terse/>
</get-interface-information>
</rpc>
```

**Comment by fbouliane** [comment]
Closed with the lastest fake-switches >= 1.1.0 and in the tag >=1.2.20
